test-worker/worker.js

```diff
@@ -70,15 +69,5 @@ exports.main = worker(async (source, rendition) => {
 }, {
     supportedRenditionFormats: SUPPORTED_FMT,
     transformerCatalog: new SenseiCatalog().catalog,
-    manifests: [
-        new Manifest({
-            "name": "sdkTestWorker",
-            "inputs": {
-                "type": ["image/jpeg", "image/png"]
-            },
-            "outputs": {
-                "type": ["image/jpeg", "image/png"]
-            }
-        })
-    ]
+    manifest: require("./pipeline-manifest.json")
```

Comment:
Ok for now - later we'll need validation.
(And a test case to verify behavior when the file is missing, not acceptable etc).

Reply:
Added a test(currently skipped), but not getting `rewire` to work (because of npm link asset-compute-pipeline, I think) 
Will come back to it when we release pipeline, unless if there is any other way to achieve rewiring..